File: Capitulo_01/01_Getting_Started:_The_Easy_Problems/02_Easy/11764_:::_Jumping_Mario/Main.c
#include "stdio.h"

int main()
{
	int casos, largo, ant, actual, subidas, bajadas, c;
	scanf("%d", &casos);
	for(c=1; c<=casos; c++)
	{
		scanf("%d", &largo);
		subidas=0; bajadas=0;

		largo--;
		scanf("%d", &ant);
		while(largo--)
		{
			scanf("%d", &actual);
			if(ant>actual) bajadas++;
			else if(ant<actual) subidas++;
			ant=actual;
		}
		printf("Case %d: %d %d\n", c, subidas, bajadas);
	}

	return 0;
}

File: Capitulo_01/01_Getting_Started:_The_Easy_Problems/02_Easy/11764_:::_Jumping_Mario/Main.java
import java.io.BufferedReader;
import java.io.File;
import java.io.FileReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main
{
	public static void main(String[] args) throws IOException
	{
		BufferedReader br=new BufferedReader(new InputStreamReader(System.in));
		//-----------------------------------------------------------
		int casos=Integer.parseInt(br.readLine()), subidas, bajadas;
		for(int c=1; c<=casos; c++)
		{
			subidas=bajadas=0;
			br.readLine();
			String[] aux=br.readLine().split(" ");

			int ant=Integer.parseInt(aux[0]);
			for(int j=1; j<aux.length ; j++)
			{
				int actual=Integer.parseInt(aux[j]);
				if(ant>actual) bajadas++;
				else if(ant<actual) subidas++;
				ant=actual;
			}
			System.out.println("Case "+c+": "+subidas+" "+bajadas);
		}
		//-----------------------------------------------------------
		br.close();
	}
}
